webapp/client/src/stories/TaxNumberPage.stories.js

```diff
@@ -150,3 +182,35 @@ WithTaxNumber.args = {
     },
   },
 };
+
+export const WithTaxNumberAndErrors = Template.bind({});
+WithTaxNumberAndErrors.args = {
+  ...Default.args,
+  fields: {
+    steuernummerExists: {
+      value: "yes",
+      errors: ["Diese Angabe fehlt."],
+    },
+    bundesland: {
+      selectedValue: "by",
+      options: [
+        { value: "bw", displayName: "Baden-Württemberg" },
+        { value: "by", displayName: "Bayern" },
+        { value: "he", displayName: "Hessen" },
+      ],
+      errors: ["Diese Angabe fehlt."],
+    },
+    bufaNr: {
+      selectedValue: undefined,
+      errors: ["Diese Angabe fehlt."],
+    },
```

Comment:
Is it on purpose that the bufanr field also has an error here although it's not shown?

Reply:
Copy-pasta 🙈